src/main/java/me/totalfreedom/tfguilds/sql/SQLDatabase.java

```diff
@@ -20,12 +22,23 @@ public SQLDatabase(TFGuilds plugin)
         }
         try
         {
-            connection = DriverManager.getConnection(String.format("jdbc:mysql://%s:%d/%s",
-                    ConfigEntry.MYSQL_HOST.getString(),
-                    ConfigEntry.MYSQL_PORT.getInteger(),
-                    ConfigEntry.MYSQL_DATABASE.getString()),
-                    ConfigEntry.MYSQL_USERNAME.getString(),
-                    password);
+            switch (ConfigEntry.CONNECTION_TYPE.getString().toLowerCase())
+            {
+                case "sqlite":
+                    connection = DriverManager.getConnection("jdbc:sqlite:" + createDBFile(plugin).getAbsolutePath().replace("%20", " "));
+                    break;
+                case "mysql":
+                    connection = DriverManager.getConnection(String.format("jdbc:mysql://%s:%d/%s",
+                            ConfigEntry.MYSQL_HOST.getString(),
+                            ConfigEntry.MYSQL_PORT.getInteger(),
+                            ConfigEntry.MYSQL_DATABASE.getString()),
+                            ConfigEntry.MYSQL_USERNAME.getString(),
+                            password);
+                    break;
+                default:
+                    connection = DriverManager.getConnection("jdbc:sqlite:" + createDBFile(plugin).getAbsolutePath().replace("%20", " "));
+            }
+
```

Comment:
This needs to be changed to prepared statements as this type of sql preparation can be vulnerable to sql injection.